engines/dreamweb/titles.cpp

```diff
@@ -99,6 +101,54 @@ void DreamWebEngine::gettingShot() {
 }
 
 void DreamWebEngine::bibleQuote() {
+	const char *enStory = "And I heard a great voice out of the temple saying to the seven angels, "
+						"Go your ways and pour out the vails of the wrath of god upon the earth. "
+						"Book of revelation Chapter 16 verse 1.";
+	const char *frStory = "Puis j'entendis une voix forte qui venait du temple et disait aux sept anges: "
+						" Allez verser sur la terre les sept coupes de la col\xC3\xA8re de Dieu."
```

Comment:
You can remove the space at the start of this  line (since there is a space at the end of the previous one).
And you need to add a space at the end of this line or at the start of the next one. Currently for me the TTS makes no pause between `Dieu` and `L'apocalypse` (which sounds strange) because of the missing space.

Reply:
The lastest update applies the principle to other versions of translation as well